git-extra/notepad

```diff
@@ -15,7 +15,12 @@ die "Usage: $0 <file>"
 
 if test -f "$1"
 then
-	unix2dos.exe "$1"
+	commitencoding="$(git config i18n.commitencoding)" ||
+		commitencoding=""
```

Comment:
Git config returns error code if the variable is not found. Unset variable (the default) means utf-8.  The OR fallback is for that case